### PR TITLE
survey form

### PR DESCRIPTION
**Pull request by manumewari**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,104 @@
...
+                <div class="col-sm-12">
+                    <button type="submit" class="btn btn-primary">Submit Details</button>
+                </div>
+            </div>
+        </div>
+    </form>
+
+</div>
+
+<!--<footer id="page-footer">A Free code camp project by Manu Mewari</footer>-->
```
Comment (sharadJay) [inline index.html]: remove comments

File: index.html [hunk 2]
```diff
@@ -1,10 +1,104 @@
...
+                    <label id="comments-label">Suggestions: </label>
+                </div>
+                <div class="col-sm">
+                    <textarea name="suggestions" rows="5" cols="30" class="form-control"
+                              placeholder="Please give your suggestions for any improvement"></textarea>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-sm-12">
```
Comment (sharadJay) [inline index.html]: if this takes up whole 12 unit then why have this class at all?

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sharadJay) [inline .idea/vcs.xml:1]: put it on gitignore

File: css/index.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+.container{
```
Comment (sharadJay) [inline css/index.css]: need space after each css path

File: index.html [hunk 5]
```diff
@@ -1,10 +1,104 @@
...
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <title>Tribute Page</title>
 </head>
 <body>
+<div id="main-container">
+    <form id="survey-form" action="">
+        <h1>Survey Form</h1>
+        <div class="container">
+            <div class="row ">
+                <div class="col-sm col-label">
```
Comment (sharadJay) [inline index.html]: why not just use col class

File: index.html [hunk 6]
```diff
@@ -1,10 +1,104 @@
...
+                <div class="col-sm">
+                    <select id="dropdown" name="City" class="form-control">
+                        <option value="delhi">Delhi</option>
+                        <option value="mumbai">Mumbai</option>
+                        <option value="bangalore">Bangalore</option>
+                        <option value="hyderabad">Hyderabad</option>
+                    </select>
+                </div>
+            </div>
+            <div class="row align-content-start">
```
Comment (sharadJay) [inline index.html:61]: how is align-content-start being used here?

File: index.html [hunk 7]
```diff
@@ -1,10 +1,104 @@
...
+                <div class="col-sm col-label">
+                    <label id="gender-label">Gender: </label>
+                </div>
+                <div class="col-sm form-check form-check-inline">
+                    <input type="radio" name="gender" id="gender1" value="M" class="form-check-input"> <label class="form-check-label" for="gender1">Male</label>&nbsp;
```
Comment (sharadJay) [inline index.html]: remove break lines, wrap it in each input and label in div

**Review by sharadJay — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (manumewari) [reply index.html]: this was for 1 space

**Review by sharadJay — APPROVED** [review]

**Pull request merged**